pages/marketplace/orders/index.tsx

```diff
@@ -30,20 +29,28 @@ import TabTitle from 'atomic/molecules/tab-title'
 const MyOrders: NextPage = () => {
   const [myOrders, setMyOrders] = useState<Order[]>([])
   const marketplaceRepository = new MarketplaceRepository()
+  const assetRepository = new AssetRepository('bscTestnet')
```

Comment:
Acho que tem ir um comentário de TODO aqui tb. Precisamos mapear esses hardcodes pq geralmente na hora que sobe pro vercel são eles que dão problema pra gente.  
```suggestion
  const assetRepository = new AssetRepository('bscTestnet')
```
```suggestion
 //TODO move to env
  const assetRepository = new AssetRepository('bscTestnet')
```